Makefile

```diff
@@ -185,6 +185,9 @@ initbench:
 bin/%.min.js: bin/%.js
 	node build/minifier.js $^ $@
 
+runtime/: src/runtime/modules/
```

Comment:
on windows when running `make prepublish` I saw the error message 
`make: *** No rule to make target 'src/runtime/modules/', needed by 'runtime/'`
I had to put this at the top of the make file 
`RUNTIME_FOLDER = src/runtime/modules` 
then updated the task to become
```sh
runtime/: $(RUNTIME_FOLDER)
	./traceur --modules=commonjs --require-runtime --dir $^ $@
```
`make prepublish` then worked for me. I guess it's a windows thang.

It's looking good. I'm looking fwd to the progress of this ;)